selfdrive/ui/tests/test_ui/run.py

```diff
@@ -21,7 +21,7 @@
 from openpilot.selfdrive.test.process_replay.vision_meta import meta_from_camera_state
 from openpilot.tools.webcam.camera import Camera
 
-UI_DELAY = 0.5 # may be slower on CI?
+UI_DELAY = 3.5 # may be slower on CI?
```

Comment:
this is a huge difference. probably from a package update?

Reply:
Even with this new delay it is failing in ci